L1Trigger/TrackFindingTracklet/python/Tracklet_cfi.py

```diff
@@ -32,4 +32,11 @@
                                                TrackQuality = cms.bool(False),
                                                TrackQualityPSet = cms.PSet(TrackQualityParams)
     )
+TTTracksFromReducedTrackletEmulation = TTTracksFromTrackletEmulation.clone(
+                                               Reduced = cms.bool(True),
+                                               # specifying where the reduced configuration files are
+                                               memoryModulesFile = cms.FileInPath('L1Trigger/TrackFindingTracklet/data/reduced_memorymodules.dat'),
```

Comment:
How will you make files data/reduced*.dat available? Will you put them in a fork of https://github.com/cms-data/L1Trigger-TrackTrigger.git ? N.B. They can go in our development branch, but should never go into central CMSSW, since the summer chain is temporary.

Reply:
See the PR in the initial comment:
https://github.com/cms-L1TK/L1Trigger-TrackFindingTracklet/pull/1